core/editable.js

```diff
@@ -1563,6 +1567,16 @@
 			// Create a new walker.
 			var walker = new CKEDITOR.dom.walker( walkerRng );
 
+			// Check if range include more than one list.
+			var multipleList = checkForMultipleList( range );
```

Comment:
I think we should rename the function, so the abowe comment could be removed. Also, the `check` in names gives no feedback about the results. The most popular conventions with `is/are`, `has`, `does` etc. are mostly the best. So we could have:  `areMultipleListInRage`, `doesRangeContainsMultipleLists` etc.